nsqadmin/package.json

```diff
@@ -10,9 +10,9 @@
     "handlebars": "1.3.0",
     "hbsfy": "1.3.2",
     "browserify": "4.1.9",
-    "gulp-sass": "0.7.1",
+    "gulp-sass": "2.0.4",
```

Comment:
I guess I had updated `node` at some point and this old version of `gulp-sass` wasn't working.